data/transactions/logic/fields.go

```diff
@@ -603,6 +618,17 @@ var globalFieldSpecs = [...]globalFieldSpec{
 	{AssetOptInMinBalance, StackUint64, modeAny, 10,
 		"The additional minimum balance required to opt-in to an asset."},
 	{GenesisHash, StackBytes32, modeAny, 10, "The Genesis Hash for the network."},
+
+	{PayoutsEnabled, StackBoolean, modeAny, incentiveVersion,
```

Comment:
Just looking at code above, the version for example AssetOptInMinBalance is hard-coded to 10 explicitly - is using a constant for the version based on feel/how much is in a particular logic sig update?

Reply:
When `AssetOptInMinBalance` was added, the thing it represented already existed, so it was definitely going in with the next avm version update.  Here, incentives are gated on when we decide to actually put them in, so all the values (and related opcodes) are gated on a named constant so it would be easy to switch to 12 if, for example, we had to rush out an 11 with some simple little update.